ddl/ddl_api.go

```diff
@@ -551,6 +555,21 @@ func checkPriKeyConstraint(col *table.Column, hasDefaultValue, hasNullFlag bool,
 	return nil
 }
 
+func checkColumnValueConstraint(col *table.Column) error {
```

Comment:
s/checkColumnValueConstraint /checkEnumColumnValue ?

Reply:
Now we should check set and enum.

Reply:
Before I comment it， it only check enum.